messages/CustomEslintEngine.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
 	"ConfigFileDoesNotExist": "The specified ESLint config file doesn't exist: %s.",
-	"InvalidJson": "Something in the ESLint config JSON is invalid. Check ESLint's JSON specifications: %s.",
+	"InvalidJson": "Something in the ESLint config JSON is invalid. Check ESLint's JSON specifications: %s %s.",
```

Comment:
The new linting rules caught that we're passing two parameters into this string even though it only accepts one. The parameters are the config file and the actual error message. I added the second parameter so that doesn't get lost.
@teresa-allen-sfdc , let me know if we need to workshop this message to properly integrate the second param.